app/theme/client/imports/general/animations.css

```diff
@@ -104,7 +104,7 @@
 	}
 }
 
-@keyframes spinh {
+@keyframes spin {
```

Comment:
Don't think this is a typo, h for horizontal (rotation across Y axis)

Reply:
ok sure will revert it